apps/api/v2/src/ee/calendars/services/gcal.service.ts

```diff
@@ -4,14 +4,14 @@ import { AppsRepository } from "@/modules/apps/apps.repository";
 import { CredentialsRepository } from "@/modules/credentials/credentials.repository";
 import { SelectedCalendarsRepository } from "@/modules/selected-calendars/selected-calendars.repository";
 import { TokensRepository } from "@/modules/tokens/tokens.repository";
+import { calendar_v3 } from "@googleapis/calendar";
```

Comment:
was this change necessary?

Reply:
@youneshenniwrites don't think this will affect anything since its just a formatting change.